sdk/storage/storage-queue/test/queueserviceclient.spec.ts

```diff
@@ -60,6 +60,59 @@ describe("QueueServiceClient", () => {
     await queueClient2.delete();
   });
 
+  it("Verify AsyncIterator(generator .next() syntax) for listQueues", async () => {
+    const queueServiceClient = getQSU();
+
+    const queueNamePrefix = getUniqueName("queue");
+    const queueName1 = `${queueNamePrefix}x1`;
+    const queueName2 = `${queueNamePrefix}x2`;
+
+    const queueClient1 = queueServiceClient.createQueueClient(queueName1);
+    const queueClient2 = queueServiceClient.createQueueClient(queueName2);
+    await queueClient1.create({ metadata: { key: "val" } });
+    await queueClient2.create({ metadata: { key: "val" } });
+
+    let iter1 = await queueServiceClient.listQueues({
+      include: "metadata",
+      prefix: queueNamePrefix
+    });
+    let queueItem = await iter1.next();
+    assert.ok(queueItem.value.name.startsWith(queueNamePrefix));
+    assert.deepEqual(queueItem.value.metadata!.key, "val");
+
+    queueItem = await iter1.next();
+    assert.ok(queueItem.value.name.startsWith(queueNamePrefix));
+    assert.deepEqual(queueItem.value.metadata!.key, "val");
+
+    await queueClient1.delete();
+    await queueClient2.delete();
+  });
+
+  it("Verify AsyncIterator(for-loop syntax) for listQueues", async () => {
```

Comment:
`marker` update doesn't get covered, can you set option `maxResults` in the option to 1 to cover that scenario?

Reply:
https://github.com/Azure/azure-sdk-for-js/blob/1c0243d28e94e65b34e28604220303d141cd24b6/sdk/storage/storage-queue/test/queueserviceclient.spec.ts#L96-L105

We are creating 4 queues and right now `maxResults=2`, which means `listQueuesSegment` gets called twice and thus `marker` update gets covered.